workflow/pipeline/imaging.py

```diff
@@ -16,16 +16,4 @@
 Session = session.Session
 Equipment = lab.Equipment
 
-
-@lab.schema
-class BrainRegion(dj.Manual):
-    definition = """
-    acronym: varchar(32)
-    ---
-    region_name: varchar(128)
-    """
-
-
-Location = BrainRegion
```

Comment:
```
Location = lab.BrainRegion
```
The `element-calcium-imaging` needs a table named `Location` in order to be activated